peewee_sanic_rest/core.py

```diff
@@ -41,7 +42,13 @@ def get_page_and_limit(self, request):
 
     async def list(self, request):
         queryset = self.get_queryset(request)
-        page, limit = self.get_page_and_limit(request)
+        page, limit, count = self.get_page_limit_count(request)
+
+        if count:
+            return json({
+                'total': await self.get_total(request, queryset)
+            })
```

Comment:
Sorry but I don't like that approach. It's counter-intuitive - when you specify `count` parameter, it ignores rest. Also, it's just `if`. You probably need custom endpoint for count, like this:

```python
    @index_route(methods=['GET'])
    async def count(self, request):
        queryset = self.get_queryset(request)
        total = await manager.count(queryset)
        return json({'total': total})
```

Later we'll probably offload pagination to specific classes, like in [DRF](http://www.django-rest-framework.org/api-guide/pagination/#custom-pagination-styles)